open_event/api/utils.py

```diff
@@ -6,6 +6,8 @@
 DEFAULT_PAGE_START = 1
 DEFAULT_PAGE_LIMIT = 20
 
+POST_RESPONSES = {401: 'Authentication failure'}
```

Comment:
Try sending a integer in string field in a post operation from Swagger UI. The api_expect returns some error code when failed. Please add it. (sorry can't run the server now)